server/services/WalletService.js

```diff
@@ -46,6 +48,12 @@ class WalletService {
 
       await this._session.commitTransaction();
 
+      await this._event.logEvent({
```

Comment:
this should be in the transaction

Reply:
Will include all the logEvent functions within the knex transactions!